addons/mail/static/src/models/thread/thread.js

```diff
@@ -2108,6 +2108,10 @@ function factory(dependencies) {
          */
         pendingSeenMessageId: attr(),
         public: attr(),
+        /**
+         * Containt the messages queued to be sent to the server by `processMessageQueue`.
+         */
+        queue: one2many('mail.message'),
```

Comment:
Need to rename to something more explicit.
The name must contain at least the following keywords (or synonyms): "messages" "pending" "send"